gui/wxpython/README.md

```diff
@@ -0,0 +1,69 @@
+# GRASS GIS - wxPython Graphical User Interface (wxGUI)
+
+## 1 - REQUIREMENTS
+
+See `../../REQUIREMENTS.md`
```

Comment:
Change as relative link? Something with at least the parentheses part like this, the label is up to choice.
```suggestion
See `[../../REQUIREMENTS.md](../../REQUIREMENTS.md)`
```